File: admin/remove_icons.py
import re

file_path = r"c:\xampp\htdocs\jain-digambar-matrimony\admin\includes\sidebar.php"

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Remove the FontAwesome <i> tags
content = re.sub(r'^\s*<i class="fas fa-[^"]*"[^>]*></i>\n?', '', content, flags=re.MULTILINE)

# Remove ml-3 from spans
content = re.sub(r'class="ml-3\s*', 'class="', content)
content = re.sub(r'\s*class=""', '', content)

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

File: admin/remove_icons.php
<?php
$file_path = "includes/sidebar.php";
$content = file_get_contents($file_path);

// Remove the FontAwesome <i> tags and potential newlines before them
$content = preg_replace('/^\s*<i class="fas fa-[^"]*"[^>]*><\/i>\r?\n?/m', '', $content);

// Remove ml-3 from spans
$content = preg_replace('/class="ml-3\s*/', 'class="', $content);
$content = preg_replace('/\s*class=""/', '', $content);

file_put_contents($file_path, $content);
echo "Icons removed successfully.";
?>
